react-common/components/controls/Feedback/configs.ts

```diff
@@ -0,0 +1,57 @@
+export const appId = 50315;
+export const feedbackFrameUrl = 'https://admin-ignite.microsoft.com';
```

Comment:
Are feedbackFramUrl and appId things we should store in targetconfig?

Reply:
This was definitely something I was considering, but these are going to be the same for each of our targets, so it feels redundant to put these in the target config of each target.

Reply:
I suppose that's true. Ideally, I think we'd want to put this somewhere that wouldn't require a release to change (i.e. what happened with our CDN) but I'm not sure if we have a good option for that without going through our backend.

Reply:
I agree this url should go in pxtarget.json, even if it is the same for each target. This is what we do for `privacyUrl`, `termsOfUseUrl`, etc.

Reply:
Somewhat related question: is the OCV/feedback feature only enabled for 1st party targets?

Reply:
Yes, we should only be showing feedback in our self-hosted targets. There is more work to really verify this, which will be my next priority, but for now, it's just something that gets set in the apptheme as something to include in the target.

Reply:
Were you able to move these hard-coded config values to pxtarget.json?